client_src/js/exceptions/octofarm-client.exceptions.js

```diff
@@ -1,13 +1,20 @@
 import { errorTypes } from "./error.types";
 
 export const ClientErrors = {
-  FAILED_VALIDATION: {
+  FAILED_VALIDATION_PATH: {
     type: errorTypes.CLIENT,
     color: "danger",
     code: "FAILED_VALIDATION_PATH",
     message: "Input validation failed, no path was supplied",
     statusCode: 412
   },
+  FAILED_VALIDATION_KEY: {
+    type: errorTypes.CLIENT,
+    color: "danger",
+    code: "FAILED_VALIDATION_KEY",
+    message: "Input validation failed, no key way supplied",
```

Comment:
way = was

The usage of the validation is not practical right now a.k.a. unusable. I dont understand the _PATH and _KEY variant or it does not meet my criteria of invalidating multiple incorrect or missing keys at once.
The backend sends back the following spec and I intend to do the same client side:
```
errors: [
 {
   "prop_name": {
      rule: "f.e. url, mongoId or wsurl",
      message: "description"
 },
 {
   "prop_name2": {
      rule: "f.e. url, mongoId or wsurl",
      message: "description"
 }, 
 ... other errors similar for other props
]
```

Maybe this explains my confusion as there is no way to pass this data and also KEY is the same as PATH to me.

Reply:
I know, needs doing in another PR they're not really relevant here and I'd realised what you said when I'd written this.

Reply:
ie... status code isn't relevant for these errors either.

Reply:
Key's are not paths so I don't get your point there

Reply:
On the key/path topic, a path error is ambiguous to me. 3 different definitions.
- an API path problem
- a file path problem 
- a child key within an JSON object like "settings:history:child_key" causing a problem 

That's how I've seen others use the `path` lingo in coding. Now looking at a key error, that's is the 3rd of those 3 definitions to me, so a dupe in my eyes.